src/org/scratchOrgSettingsGenerator.ts

```diff
@@ -209,62 +229,92 @@ export default class SettingsGenerator {
     }
   }
 
-  private createPackageXml(apiVersion: string): void {
-    const pkg = `<?xml version="1.0" encoding="UTF-8"?>
-<Package xmlns="http://soap.sforce.com/2006/04/metadata">
-    <types>
-        <members>*</members>
-        <name>Settings</name>
-    </types>
-    <types>
-        <members>*</members>
-        <name>CustomObject</name>
-    </types>
-    <version>${apiVersion}</version>
-</Package>`;
-    this.writer.addToZip(pkg, path.join(this.shapeDirName, 'package.xml'));
-  }
-
-  private createObjectFileContent(json: Record<string, unknown>) {
-    const output: ObjectSetting = {};
-    if (json.sharingModel) {
-      output.sharingModel = upperFirst(json.sharingModel as string);
+  private createObjectFileContent(
+    name: string,
+    json: Record<string, unknown>,
+    allRecordTypes: string[],
+    allbusinessProcesses: string[]
+  ) {
+    let output = {
+      '@': {
+        xmlns: 'http://soap.sforce.com/2006/04/metadata',
+      },
+    } as JsonMap;
+    const sharingModel = json['sharingModel'];
```

Comment:
is a string accessor required here? why not `json.sharingModel` anymore? Or extend the type definition with 
`json: Record<string, unknown> & { sharingModel?: string },` and avoid casting it `as string`